composer.json

```diff
@@ -18,7 +18,9 @@
         "atoum/stubs": "^2.2",
         "brick/geo": ">=0.5 <=1.0",
         "symfony/expression-language": "^6.3 || ^7.2",
+        "symfony/serializer": "^6.0 || ^7.0",
         "symfony/security-bundle": "^6.0 || ^7.0",
+        "symfony/security-core": "^6.0 || ^7.0",
```

Comment:
fun story: security-bundle@6.0 requires security-core ^5.4 | ^6.0 ; tests were perfectly fine when I added the User provider, even with lowest deps.
But it should not have been this way: the interface for the UserProvider was different in security-core 5.4 (loadByUsername became loadByIdentifier in 6.0). [Tests were green](https://github.com/ccmbenchmark/ting_bundle/actions/runs/12809980381/job/35715974943) even with security-core 5.4 ; but not anymore. So we lock the core at 6.0 minimum to avoid that.